core/src/test/java/io/debezium/connector/cassandra/TestUtils.java

```diff
@@ -142,9 +141,8 @@ public static String keyspaceTable(String tableName) {
      * Generate a commit log file with current timestamp in memory
      */
     public static File generateCommitLogFile() {
-        File cdcDir = new File(DatabaseDescriptor.getCDCLogLocation());
         long ts = System.currentTimeMillis();
-        return Paths.get(cdcDir.getAbsolutePath(), "CommitLog-6-" + ts + ".log").toFile();
+        return Paths.get("/tmp", "CommitLog-6-" + ts + ".log").toFile();
```

Comment:
`/tmp` os OS specific, you can use something like `Files.createTempDirectory("cassandra_test_").toString()` to make it OS agnostic (also is more safe as it should create different dir for each test).

Reply:
it really does not do anything, it does not matter. it will not place any files there and so on ... it is irrelevant how it is called.

Reply:
well, but it's a generic test utility which may be used in the future in other places and in a wrong way. We either should generate existing path (which would work on every OS) or return just a constant or random string and name the method accordingly. IMHO the best at this point would be to remove this method completely and use some string (e.g. `non-existing-log-file-path`) directly in the test.

Reply:
@vjuranek  fixed and pushed, lets see.